agent/consul/server.go

```diff
@@ -1021,6 +1047,10 @@ func (s *Server) GetLANCoordinate() (lib.CoordinateSet, error) {
 
 // GetWANCoordinate returns the coordinate of the server in the WAN gossip pool.
 func (s *Server) GetWANCoordinate() (*coordinate.Coordinate, error) {
+	if s.serfWAN == nil {
+		// Return zero values if WAN federation is disabled
+		return &coordinate.Coordinate{}, nil
```

Comment:
Is this the right thing to do vs `ErrWANFederationDisabled` may well be as I'm not familiar with the ways it is called but it's not obvious to me why we'd want to pretend this still works when WAN is disabled.